Standards for developing flexible, durable, and sustainable HTML and CSS.

```diff
@@ -0,0 +1 @@
+http://codeguide.co/
```

Comment:
nice addition, but add it to free-programming-books.md (39), not as a separate file